README.md

```diff
@@ -1 +1,51 @@
-<!-- Just Created for now to push changes to remote repo -->
\ No newline at end of file
+# Capstone Project
+
+> This project contains the HTML, CSS and JavaScript Capstone Project. Capstone Projects are done at the end of a module at Microverse.
+
+
+# Project Name
+## StartUp Lab Summit
+
+** StartUp Lab is an annual conference event that is held internationally for individuals to participate in real life and join us live on YouTube. Focus of this annual event is to address individuals, who wants to start their own business or have already started. ** 
+
+## Live Demo Link for Github-Page Branch
+
+![Click Here to view presentation video](https://www.loom.com/share/44ab3ae3582e4eb6aa25ce0dfff240b5)
+
+[Click Here to view live version](https://faizi2500.github.io/Module1-Capstone-Project/)
+## Built With
+### Major languages
+- HTML
+- CSS
+- JavaScript
+- Linters
+
+### Technologies used
+- Github
+- Git
+- Visual Studio Code
+- Ubuntu
+- Photoshop
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+git clone https://github.com/faizi2500/Module1-Capstone-Project.git
+
+
+## Authors
+👤 Faizan Zahid
+
+- GitHub: [@faizi2500 ](https://github.com/faizi2500)
+- Twitter: [@faizi_250 ](https://twitter.com/Faizy_250) 
+- LinkedIn: [@faizan2500](www.linkedin.com/in/faizan2500)
+
+## 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+## Show your support
+Give a ⭐️ if you like this project!
+
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ] Add an "Acknowledgements" section or similar that gives appropriate credit to Cindy Shin the author of the original design, as required by the Creative Commons License.